src/core/namespace.c

```diff
@@ -1216,7 +1216,7 @@ static int mount_image(const MountEntry *m, const char *root_directory) {
         }
 
         r = verity_dissect_and_mount(
-                                mount_entry_source(m), mount_entry_path(m), m->image_options,
+                                -1 /* src_fd */, mount_entry_source(m), mount_entry_path(m), m->image_options,
```

Comment:
we usually write these commands like this:

```c
…, /* src_fd= */ -1, …
```

i.e. write them before the argument, and sufficed by `=`